taca/server_status/server_status.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import json
+import gspread
+from oauth2client.client import SignedJwtAssertionCredentials as GCredentials
+from taca.utils.config import CONFIG
+
+def get_disk_space():
+	result = {}
+	config = CONFIG['server_status']
+	servers = config.get('servers', [])
+	for server_url in servers.keys():
+		# get path of disk
+		path = servers[server_url]
+
+		# get command
+		command = config['command'] + ' ' + path
+
```

Comment:
I think this syntax is preferred
`command = "{} {}".format(config['command'], path)`

right @Galithil ?